src/pages/commits/ActiveCommits/CommitChartLabel.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Disclaimer } from "@leafygreen-ui/typography";
+import { format } from "date-fns";
```

Comment:
We have a wrapper util function around this. That should handle the date and time formatting you do below. https://github.com/evergreen-ci/spruce/blob/6cf75a90683e6bb58e5bccd9ebce41f4683e9675/src/utils/string/index.ts

Reply:
Really helpful, thank you!